pages/login.page.js

```diff
@@ -0,0 +1,29 @@
+const PAGE_TIMEOUT = 10000;
+
+class LoginPage {
+    
+    get usernameInput () { return $('#username'); }
+    get passwordInput () { return $('#password'); }
+    get signInButton () { return $('#submit'); }
+    get rememberMe () { return $('#remember-me'); }
+    get alert () {return $('[class= "badge badge-pill badge-danger"]'); }
+
+
+    async logIn(username, password) {
+        await this.usernameInput.waitForClickable({ timeout: PAGE_TIMEOUT });
+        await this.usernameInput.setValue(username);
+        await this.passwordInput.waitForClickable({ timeout: PAGE_TIMEOUT });
+        await this.passwordInput.setValue(password);
+    }
+
+    async open(link) {
+        await browser.url(`${link}`);
```

Comment:
As you are not interpolating a string here, but rather just passing a raw string directly to `browser.url()`, there's no need for **template literal** syntax. You can just do: `await browser.url(link)` to simplify it